lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -244,6 +241,8 @@ def filter_stream_from_files(tag, es)
         end
       end
 
+      metadata['kubernetes']['container_name'] = match_data['container_name'] if match_data
```

Comment:
Sorry - I thought I was only approving individual commits, not the entire PR.  Not used to the github approval process yet.